### PR TITLE
[3102] - Move mail notifications to background jobs

### PR DESCRIPTION
**Pull request by TobyRet**
### Context
- We have a new Sidekiq background container running, similar to geocoding.

https://trello.com/c/jb4y3epg/3102-m-background-mail-notifications

### Changes proposed in this pull request
- Mail notifications are now sent via new background jobs.

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

File: app/jobs/send_welcome_job.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class SendWelcomeJob < ApplicationJob
+  queue_as :mailer
```
Comment (misaka) [inline app/jobs/send_welcome_job.rb]: Would it be worth syncing up the name of the Docker container and this? It's called `bg-emailer` so maybe `bg-mailer`? Or change the queue name here to `mailer`? Just seems like a cheap, but minor, win.

**Comment by avinhurry** [comment]
[Searching for an accredited body no longer shows all courses they are the awarding body for](https://trello.com/c/1ZmfMZUo/3141-searching-for-an-accredited-body-no-longer-shows-all-courses-they-are-the-awarding-body-for)

**Comment by SamJamCul** [comment]
[[M] Background mail notifications](https://trello.com/c/jb4y3epg/3102-m-background-mail-notifications)

**Comment by SamJamCul** [comment]
[[M] Background mail notifications](https://trello.com/c/jb4y3epg/3102-m-background-mail-notifications)

File: app/jobs/send_welcome_job.rb (hunk 1 above)
Comment (SamJamCul) [reply app/jobs/send_welcome_job.rb]: Done!

**Comment by TobyRet** [comment]
[[M] Background mail notifications](https://trello.com/c/jb4y3epg/3102-m-background-mail-notifications)

**Comment by TobyRet** [comment]
Just saw Graham's comment. Going to reopen PR shortly

**Pull request closed without merging**